flocker/provision/_effect.py

```diff
@@ -0,0 +1,56 @@
+import sys
+from characteristic import attributes
+from effect import (
+    sync_performer, Effect,
+    ComposedDispatcher, TypeDispatcher, base_dispatcher)
+from effect.do import do, do_return
+
+
+# This should be upstream.
```

Comment:
https://github.com/radix/effect/pull/46

For the moment, I'm going to avoid addressing issues in this file, in anticipation that it will be thrown away soon.